modules/LiteEvent.ts

```diff
@@ -1,24 +0,0 @@
-//http://stackoverflow.com/questions/12881212/does-typescript-support-events-on-classes
```

Comment:
Yeah I wanted to remove this but wasn't 100% on whether we were going to keep the P/S at the time